_tools/release_cycle_template.svg.jinja

```diff
@@ -58,25 +58,87 @@
         <!-- Colourful blob with a label -->
         {% set start_x = date_to_x(version.first_release_date) %}
         {% set end_x = date_to_x(version.end_of_life_date) %}
-        {% set mid_x = (start_x + end_x) / 2 %}
-        <rect
-            class="release-cycle-blob release-cycle-blob-{{ version.status }}"
-            x="{{ start_x * SCALE }}"
-            y="{{ (y - 1) * SCALE }}"
-            width="{{ (end_x - start_x) * SCALE }}"
-            height="{{ 1.25 * SCALE }}"
-            rx="0.25em"
-            ry="0.25em"
-        />
-        <text
-            class="release-cycle-blob-label release-cycle-blob-{{ version.status }}"
-            x="{{ mid_x * SCALE }}"
-            y="{{ (y - 0.1) * SCALE }}"
-            font-size="{{ SCALE * 0.75 }}"
-            text-anchor="middle"
-        >
-            {{ version.status }}
-        </text>
+
+        {% if version.status == "bugfix" %}
+            <!-- bugfix status needs a security tail.
+                Draw the rectangle with two path elements instead.
+                Thanks Claude.ai for the conversion.
+            -->
+            {% set half_x = date_to_x(version.start_security_date) %}
+            {% set height = 1.25 * SCALE %}
+            {% set left_width = (half_x - start_x) * SCALE %}
+            {% set right_width = (end_x - half_x) * SCALE %}
+            {% set left_x = start_x * SCALE %}
+            {% set middle_x = half_x * SCALE %}
+            {% set right_x = half_x * SCALE %}
+            {% set recty = (y - 1) * SCALE %}
+            {% set radius_value = 0.25 * SCALE %}
+
+            <path
+                class="release-cycle-blob release-cycle-blob-bugfix"
+                d="
+                    M{{ left_x + radius_value }},{{ recty }}
+                    Q{{ left_x }},{{ recty }} {{ left_x }},{{ recty + radius_value }}
+                    V{{ recty + height - radius_value }}
+                    Q{{ left_x }},{{ recty + height }} {{ left_x + radius_value }},{{ recty + height }}
+                    H{{ middle_x }}
+                    V{{ recty }}
+                    Z
```

Comment:
I find this rather confusing.
Once the final style is decided, would you mind me simplifying & adding comments? Unless you want to do it of course.